packages/array-standard-deviation/src/index.js

```diff
@@ -0,0 +1,11 @@
+import variance from 'ml-array-variance';
+
+/**
+ * Computes the standard deviation of the given values
+ * @param {Array} values
+ * @param {boolean} [unbiased = true] - if true, divide by (n-1); if false, divide by n.
+ * @return {number}
+ */
+export default function standardDeviation(values, unbiased) {
```

Comment:
I know it was like that before, but ml-stat is an old library and doesn't follow our current rules